task/06-conditions-n-loops-tasks.js

```diff
@@ -81,8 +95,9 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,1,1   =>  false
  *   10,10,10 =>  true
  */
-function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+function isTriangle(a, b, c) {
+    if (a + b > c && a + c > b && b + c > a) return true;
+    else return false;
```

Comment:
Please, solve this task using only one return (a + b > c && a + c > b && b + c > a)
https://javascript.info/logical-operators